frontend/src/containers/AddChart.tsx

```diff
@@ -190,6 +192,10 @@ function AddChart() {
             values.chartType === ChartType.DonutChart) && {
             dataLabels: values.dataLabels,
           }),
+          ...((values.chartType === ChartType.PieChart ||
```

Comment:
Same here, is PartToWhole part of this PR?

Reply:
No, I missed one reference that I should have deleted